gammapy/data/event_list.py

```diff
@@ -354,6 +355,35 @@ def filter_circular_region(self, region):
             mask = np.union1d(mask, temp)
         return mask
 
+    def fill_skymap(self, skymap):
+        """Fill events in sky map
+
+        Parameters
+        ----------
+        skymap : `~gammapy.image.SkyMap`
+            Sky map to fill
+
+        Returns
+        -------
+        filled_map : `~gammapy.image.SkyMap`
+            Sky map with events filled
+        """
+        from gammapy.image import SkyMap
+
+        shape = skymap.data.shape
+
+        coords = skycoord_to_pixel(self.radec, skymap.wcs)
```

Comment:
Filling counts images and cubes is implemented and pretty well tested.
See e.g. here:
https://github.com/gammapy/gammapy/blob/master/gammapy/scripts/tests/test_image_bin.py#L15

I'm +1 to use SkyMap and make better APIs, but you have to take the time and remove the other functions and update the tests.